Changes

```diff
@@ -768,8 +767,17 @@ OCaml 4.12.0
 - #10048: Fix bug with generalized local opens.
   (Leo White, review by Thomas Refis)
 
-OCaml 4.11.1
-------------
+OCaml 4.10 maintenance branch
```

Comment:
4.10 => 4.11

Reply:
Thanks! I fixed this and would like to merge when the CI is happy.